lib/contentful_delivery/delivery.ex

```diff
@@ -375,9 +384,9 @@ defmodule Contentful.Delivery do
 
       "foobar" = Contentful.Delivery.config(:my_config)
   """
-  @spec config(atom(), any() | nil) :: any()
-  def config(setting, default \\ nil) do
-    config() |> Keyword.get(setting, default)
```

Comment:
Any specific reason to not use Keyword.get/3 with default anymore?

Reply:
It's an outcome from the BEAM V conference, [a talk](https://www.youtube.com/watch?v=Qm920yYm--U) by @expede. She advocated this approach which makes the code overall more readable imho, since I do not have to remember the parameter order of `Keyword.get`.

I wanted to try it - and then decide later if I like it and keep it once I extract the Configuration/Request handling from the `Contentful.Delivery` module.